### DISCUSSION TITLE
Windows 11 Desktops - multiple application instances

### DISCUSSION THREAD

**Discussion by stewartbourke487** [sumatrapdfreader/sumatrapdf#2384] [Q&A]
Are there any plans to formally support Sumatra PDF in multiple desktops, with different documents open in each desktop?  I have read some suggestions about copying instances of the .exe etc but they are not ideal solutions.   

The application is an excellent reader, but having to have the same documents open across multiple desktop defeats one of the advantages of multiple desktops.

The requirement is to allow multiple instances of the application be open, and for each instance to have its own tabset.  

Thank you.

**Comment by kjk** [comment]
You can create a new Sumatra window with Ctrl-N and drag it to another desktop so as far I can tell, Sumatra already supports that.
